launcher/image/test/scripts/test_experiment_value.sh

```diff
@@ -0,0 +1,21 @@
+ #!/bin/bash
+set -euo pipefail
+source util/read_serial.sh
+
+# This test requires the workload to run and print
+# corresponding messages to the serial console.
+SERIAL_OUTPUT=$(read_serial $2 $3) 
+print_serial=false
+
+if echo $SERIAL_OUTPUT | grep -q "EnableTestFeatureForImage:$1"
+then
+    echo "- test experiment verified $1"
+else
+    echo "FAILED: experiment status expected to be $1"
+    echo "TEST FAILED. Test experiment status expected to be $1" > /workspace/status.txt
+    print_serial=true
```

Comment:
very nit, but you could just `echo $SERIAL_OUTPUT` here instead of have the `print_serial` var

Reply:
Cleaned up VMs for now and restarted these tests.
Will look into the quota now.

Reply:
Yes definitely, Ill handle this in a follow-up CL. We do this in all of our scripts and we should clean them all up together, IMO.
PLMK if you disagree and I can handle this now.